### ISSUE TITLE
Add classname column in table output of instance names

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbemtools#259]
Instance names in table output format are currently shown with host, namespace and keybindings, and the class name is missing. Because some operations can return instances of different classes, it is important to show the class name.

Example of today's output:
```
$ pywbemcli -m tests/unit/simple_mock_model_ext.mof -o table instance enumerate CIM_Foo -o
InstanceNames: CIM_Foo
+--------+-------------+------------------------------------------------+
| host   | namespace   | keybindings                                    |
|--------+-------------+------------------------------------------------|
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo1'})         |
. . .
+--------+-------------+------------------------------------------------+
```

**Comment by KSchopmeyer** [comment]
See issue # 258 and proposed solution.  I agree that this makes more sense.

**Comment by andy-maier** [comment]
I agree with adding the class name as shown in PR #258.

**Comment by andy-maier** [comment]
Addressed in PR #273
